modules/optflow/src/rlofflow.cpp

```diff
@@ -281,6 +281,8 @@ class SparseRLOFOpticalFlowImpl : public SparseRLOFOpticalFlow
         CV_Assert(!prevImg.empty() && prevImg.depth() == CV_8U && (prevImg.channels() == 3 || prevImg.channels() == 1));
         CV_Assert(!nextImg.empty() && nextImg.depth() == CV_8U && (nextImg.channels() == 3 || nextImg.channels() == 1));
         CV_Assert(prevImg.sameSize(nextImg));
+        if ((param->supportRegionType == SR_CROSS) && (prevImg.channels() != 3 || nextImg.channels() != 3))
```

Comment:
Need to check for empty `param` here to avoid crash.
Will add PR soon.

/cc @tsenst